### PR TITLE
Add readme for image-builder

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Adding readme for image-builder

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/aws/image-builder/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    * 2 vCPUs
+    * 8 GB RAM
+* Machine must have network access to
+    * vCenter endpoint
+    * public.ecr.aws (to download container images from EKS-A)
+    * anywhere-assets.eks.amazonaws.com (to download EKS-A binaries)
+    * distro.eks.amazonaws.com (to download EKS-D binaries)
+    * d2glxqk2uabbnd.cloudfront.net (for EKS-A and EKS-D ECR container images)
+
+#### Building a vSphere OVA Node Image
```
Comment (chrisnegus) [inline projects/aws/image-builder/README.md:131]:
```suggestion
#### Building a baremetal Node Image
```
Is that what you meant?

File: projects/aws/image-builder/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+sudo usermod -a -G kvm ubuntu
+sudo chmod 666 /dev/kvm
+sudo chown root:kvm /dev/kvm
+```
+2. Build or download the image builder tool
+3. Run the image builder tool for appropriate release channel
+```
+image-builder build --os ubuntu --hypervisor baremetal --release-channel <release channel>
+```
+
```
Comment (chrisnegus) [inline projects/aws/image-builder/README.md:146]: Might want to point to the bare metal docs to show them how to set up the image to include include it in a cluster create.

**Review by chrisnegus — COMMENTED** [review]
I have a couple of small suggestions.

File: projects/aws/image-builder/README.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  "password":"<vcenter username>",
+  "resource_pool":"<resource pool used for image building vm>",
+  "username":"<vcenter username>",
+  "vcenter_server":"<vcenter fqdn>",
+  "vsphere_library_name": "<vsphere content library name>"
+}
+```
+5. Run the image builder tool for appropriate release channel
+```
+image-builder build --os ubuntu --hypervisor vsphere --vsphere-config <path to above json file> --release-channel <release channel>
```
Comment (jaxesn) [inline projects/aws/image-builder/README.md:110]: release channel being `1-20` for ex? i know you defined them above but i was confused at first read, not sure if there is a way to spell it out better here?

Comment (vignesh-goutham) [reply projects/aws/image-builder/README.md:110]: yea I can put in an example here

File: projects/aws/image-builder/README.md (hunk 1 above)
Comment (vignesh-goutham) [reply projects/aws/image-builder/README.md:131]: yea, good catch

**Review by chrisnegus — COMMENTED** [review]
/lgtm

**Review by chrisnegus — APPROVED** [review]

**Pull request merged**